src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/util/AuthorizationEnabledIntegrationTest.java

```diff
@@ -448,6 +451,51 @@ public void userProfileCreateUserWireMock(HttpStatus status) {
         );
     }
 
+    public void userProfileCreateUserWireMockWithExtraRoles() {
+
+        String body = null;
+        body = "{"
+                + "  \"idamId\":\"" + UUID.randomUUID().toString() + "\","
+                + "  \"idamRegistrationResponse\":\"201\""
+                + "}";
+
+        userProfileService.stubFor(post(urlEqualTo("/v1/userprofile"))
+                .willReturn(aResponse()
+                        .withHeader("Content-Type", "application/json")
```

Comment:
Can you please try import io.restassured.http.ContentType.json.name, or anything similar?
I can see about 14 instances of this String in file, which can be replaced.

Reply:
This has been addressed here: https://github.com/hmcts/rd-professional-api/pull/961/commits/5f7434ab43892d85fe65a367945026340e025f0b